includes/definitions/moxa-etherdevice.yaml

```diff
@@ -11,3 +11,4 @@ mib_dir:
 discovery:
     - sysObjectId:
         - .1.3.6.1.4.1.8691.7.116
+        - .1.3.6.1.4.1.8691.7.69
```

Comment:
You will need to add tests for this as well: https://docs.librenms.org/#Developing/os/Test-Units/